logstash-input-eventlog.gemspec

```diff
@@ -24,10 +24,7 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'logstash-codec-plain'
 
-  if RUBY_PLATFORM == 'java'
```

Comment:
When installing the plugin locally, i am getting the following exception:

```
Plugin version conflict, aborting
ERROR: Installation Aborted, message: Bundler could not find compatible versions
 for gem "logstash-core":
  In snapshot (Gemfile.lock):
    logstash-core (= 2.0.0.rc1)

  In Gemfile:
    logstash-input-elasticsearch (>= 0) java depends on
      logstash-core (< 3.0.0, >= 2.0.0.beta2) java
```

Need to modify the runtime dependency to:

```
  s.add_runtime_dependency "logstash-core", ">= 2.0.0.beta2", "< 3.0.0"
```